packages/fiori/src/WizardTab.js

```diff
@@ -0,0 +1,289 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { isSpace, isEnter } from "@ui5/webcomponents-base/dist/Keys.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+
+import WizardTabTemplate from "./generated/templates/WizardTabTemplate.lit.js";
+import WizardTabCss from "./generated/themes/WizardTab.css.js";
+
+const metadata = {
+	tag: "ui5-wizard-tab",
+	properties: /** @lends sap.ui.webcomponents.fiori.WizardTab.prototype */ {
+		/**
+		 * Defines the <code>icon</code> of the step.
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		icon: {
+			type: String,
+		},
+
+		/**
+		 * Defines the <code>heading</code> of the step.
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		heading: {
+			type: String,
+		},
+
+		/**
+		 * Defines the <code>subheading</code> of the step.
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		subheading: {
+			type: String,
+		},
+
+		/**
+		 * Defines initials to be displayed if no <code>icon</code> is set.
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		initials: {
```

Comment:
I think this should be called something else, as "initials" is more suitable for avatar, where we display the initials of a person's name. This could be probably "text" or "number"

Reply:
I agree, "number" is better.